build/tools/CFamilyJarPreProcessor/Preprocessor.cs

```diff
@@ -36,6 +36,7 @@ internal class Preprocessor
         {
                 "Sonar_way_profile.json",
                 "RulesList.json",
+                "MisraRulesList.json",
```

Comment:
Update the pre-processor so it fetches the additional files we need:
* `MisraRulesList.json`, which lists all of the MISRA rules (not whether they are active or not)
* the rule .json files form the `misra23` subfolder (line #49 below)